pallets/polkadex-ido/src/lib.rs

```diff
@@ -384,7 +381,7 @@ pub mod pallet {
                         <InfoFundingRound<T>>::remove(&round_id);
                         Self::deposit_event(Event::CleanedupExpiredRound(round_id));
                     }
-                }
+                } */
             }
```

Comment:
I assume the whole block can be commented out. There is no need to loop thru here.

Reply:
Updated ✅